src/Symfony/Bridge/PhpUnit/README.md

```diff
@@ -0,0 +1,35 @@
+PHPUnit Bridge
+==============
+
+Tools for helping with testing PHP projects, especially useful to deal with user
+deprecation notices. Should be used only in the `require-dev` section of your
+composer.json file (not in `require`).
+
```

Comment:
`Provides utilities for PHPUnit like deprecation notices management.`

And I would remove everything else. The `require-dev` section sentence should be move in another paragraph that explains how to use the bridge in a project.